arangod/Aql/ExecutionPlan.cpp

```diff
@@ -406,10 +418,30 @@ std::unique_ptr<graph::BaseOptions> createShortestPathOptions(
         TRI_ASSERT(value->isConstant());
 
         if (name == "weightAttribute" && value->isStringValue()) {
-          options->setWeightAttribute(
-              std::string(value->getStringValue(), value->getStringLength()));
+          options->setWeightAttribute(value->getString());
         } else if (name == "defaultWeight" && value->isNumericValue()) {
           options->setDefaultWeight(value->getDoubleValue());
+        } else if (name == StaticStrings::IndexHintOptionForce) {
+#ifndef ENABLE_FORCED_INDEX_HINTS_FOR_GRAPH_OPERATIONS
```

Comment:
Is this macro completely implemented or shall we just drop it? Or do you just want to keep it (which is fine by me)